src/main/java/core/basesyntax/service/FileWriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriterService {
+    void writeToCsvFile(String report, String destination);
```

Comment:
```suggestion
    void write(String report, String destination);
```
There is no specific logic for processing csv files in the reader and the writer
Those services just read data, and write some data to files